### PR TITLE
Shiree product details page

### PR DESCRIPTION
**Pull request by ShireeWill**
edited also small changes in header and footer

### CODE REVIEW

File: src/pages/product-page/product-page.module.scss [hunk 1]
```diff
@@ -8,20 +8,57 @@
...
+    padding-right: 30px;
+    padding-left: 30px;
+    padding-top: 30px;
+    padding-bottom: 60px;
 }
 
 .left {
     grid-auto-flow: row;
-    flex: 1;
+    flex-basis: 1px;
```
[the remark this answers is not in the document]
Comment (alissawix) [reply src/pages/product-page/product-page.module.scss:19]: what were you trying to do?

File: src/pages/product-page/product-page.module.scss [hunk 2]
```diff
@@ -8,20 +8,57 @@
...
+    width: 140px;
+    margin-top: 20px;
+    height: 47px;
+}
+
+.productInfo {
+    display: flex;
+}
+
+.productInfo {}
```
Comment (alissawix) [inline src/pages/product-page/product-page.module.scss:48]: delete

File: src/pages/product-page/product-page.tsx [hunk 3]
```diff
@@ -39,25 +42,38 @@ export const ProductPage: React.FC<ProductPageProps> = ({ className }) => {
     }
     return (
         <div className={classNames(styles.root, className)}>
-            <ProductImages
-                mainImage={product.media?.mainMedia}
-                images={product.media?.items}
-                className={styles.left}
-            />
+            <div className={styles.leftPanel}>
```
[the remark this answers is not in the document]
Comment (alissawix) [reply src/pages/product-page/product-page.tsx:45]: specially that it has a class

File: src/pages/product-page/product-page.tsx [hunk 4]
```diff
@@ -39,25 +42,38 @@ export const ProductPage: React.FC<ProductPageProps> = ({ className }) => {
...
-                    <input className={commonStyles.numberInput} type="number" placeholder="1" />
-                </label>
-                <button
-                    onClick={() => wixApi.addToCart(product._id!)}
-                    className={commonStyles.primaryButton}
-                >
-                    Add to Cart
-                </button>
-                <ProductInfo />
+                <div className={styles.productForm}>
```
Comment (alissawix) [inline src/pages/product-page/product-page.tsx:53]: why another div?

File: src/pages/product-page/product-page.tsx [hunk 5]
```diff
@@ -39,25 +42,38 @@ export const ProductPage: React.FC<ProductPageProps> = ({ className }) => {
...
+                    <div className={styles.productTitle}>{product.name}</div>
+                    {product.price && (
+                        <div className={CartItem_module.price}>
```
Comment (alissawix) [inline src/pages/product-page/product-page.tsx:56]: the class comes from cart item..?

File: src/pages/product-page/product-page.tsx [hunk 6]
```diff
@@ -8,6 +8,9 @@ import { WixAPIContext } from '../../api/WixAPIContextProvider';
 import commonStyles from '../../styles/common-styles.module.scss';
 import { ProductImages } from './product-images/product-images';
 import { ProductInfo } from './product-info/product-info';
+import ProductInfo_module from './product-info/product-info.module.scss';
```
Comment (alissawix) [inline src/pages/product-page/product-page.tsx:11]: why do we import all these styles?

**Review by alissawix — CHANGES_REQUESTED** [review]

File: src/pages/product-page/product-page.module.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShireeWill) [reply src/pages/product-page/product-page.module.scss:19]: ### Outdated.

File: src/pages/product-page/product-page.tsx (hunk 5 above)
Comment (ShireeWill) [reply src/pages/product-page/product-page.tsx:56]: yes

File: src/pages/product-page/product-page.tsx (hunk 6 above)
Comment (ShireeWill) [reply src/pages/product-page/product-page.tsx:11]: Didn't clean the files yet. Probabley from old project..

File: src/pages/product-page/product-page.tsx (hunk 5 above)
Comment (alissawix) [reply src/pages/product-page/product-page.tsx:56]: move to common

File: src/pages/product-page/product-page.tsx (hunk 6 above)
Comment (alissawix) [reply src/pages/product-page/product-page.tsx:11]: stop using them and delete

**Review by alissawix — APPROVED** [review]

**Pull request merged**